README.md

```diff
@@ -162,6 +162,22 @@ $ ruby start_finish_trace.rb
 
 **Busted** includes an [example `dtrace` probe](/dtrace/probes/examples/method-cache-clear.d) for use on the command line or an application.  See the [probe](/dtrace/probes/examples/method-cache-clear.d) for usage.
 
+## Rails Usage
+
+In rails you can use the `BustedTracer` middleware to log the method and constant cache invalidations. You can configure BustedTracer using,
+
+```ruby
+config.middleware.insert_after(ActionDispatch::Static, BustedTracer)
```

Comment:
Do you mean test if it works first in the middleware stack, or specifically require the middleware to be first in the stack?